vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -595,9 +596,9 @@ class RoomDetailFragment @Inject constructor(
                 addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION or Intent.FLAG_ACTIVITY_NEW_TASK)
             }
 
-            if (intent.resolveActivity(requireActivity().packageManager) != null) {
```

Comment:
What about the 2 other usages of `resolveActivity` ?

Reply:
I did not change those as I don't have any idea how they're being used. I see that one usage of it for VectorWebViewClient and the other usage is for crypto. I honestly have no idea about crypto. I've been trying to learn the mvrx and epoxy library more so I can understand the overall code of this app better. If you could let me know how to test out the other usages of `resolveActivity`, I would love to help out by digging into it more.

Reply:
Applying the same fix at the 2 other places should be fine. Can you do that please?